Resources/doc/4-integrating_fosub.md

```diff
@@ -0,0 +1,188 @@
+Step 4: Integrating with FOSUserBundle
+======================================
+
+If you use FOSUserBundle and you want to integrate it with HWIOAuthBundle this article will describe most mahor
+FOSUserBundle is the most popular Symfony2 bundle, and when you use HWIOAuthBundle for allow to users to authenticate
```

Comment:
`...when you use HWIOAuthBundle to authenticate via different networks and services, you...`